Can't install when using latest node lts(v18)

### Describe the bug
```
    ⠏ Installing npm dependencies (wow these are heavy)
/home/linux/.npm/_npx/e31027f3785124a8/node_modules/gluegun/build/index.js:15
    throw up;
    ^
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: azarna@0.0.1
npm ERR! Found: expo@48.0.20
npm ERR! node_modules/expo
npm ERR!   expo@"^48.0.15" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer expo@"^49.0.7" from @expo/webpack-config@18.1.3
npm ERR! node_modules/@expo/webpack-config
npm ERR!   @expo/webpack-config@"^18.0.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/linux/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/linux/.npm/_logs/2023-09-17T21_11_28_937Z-debug-0.log

(Use `node --trace-uncaught ...` to show where the exception was thrown)
```
### Ignite version

8.8.7

### Additional info

System
  platform           linux                                                  
  arch               x64                                                    
  cpu                12 cores     Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz 
  directory          Azarna       /home/projects/Azarna                     

JavaScript (and globally-installed packages)
  node               18.17.1      /home/linux/.nvm/versions/node/v18.17.1/bin/node 
  npm                9.6.7        /home/linux/.nvm/versions/node/v18.17.1/bin/npm  
    corepack         0.18.0                                                        
    eas-cli          5.2.0                                                         
    npm              9.6.7                                                         
  yarn               -            not installed                                    
  pnpm               -            not installed                                    
  expo               -            not installed                                    

Ignite
  ignite-cli         8.8.7        /home/linux/.npm/_npx/e31027f3785124a8/node_modules/.bin/ignite      
  ignite src         build        /home/linux/.npm/_npx/e31027f3785124a8/node_modules/ignite-cli/build 

Android
  java               11.0.20.1    /usr/bin/java 
  android home       -            undefined     

Tools
  git                git version 2.25.1   /usr/bin/git

--- a/boilerplate/package.json
+++ b/boilerplate/package.json
@@ -33,7 +33,7 @@
   "dependencies": {
     "@expo-google-fonts/space-grotesk": "^0.2.2",
     "@expo/metro-config": "^0.7.1",
-    "@expo/webpack-config": "^18.0.1",
+    "@expo/webpack-config": "18.1.2",
     "@react-native-async-storage/async-storage": "1.17.11",
     "@react-navigation/bottom-tabs": "^6.3.2",
     "@react-navigation/native": "^6.0.2",

